pkg/pipeline/utils/prom_server.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package utils
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+)
+
+// StartServer listens for prometheus resource usage requests
+func StartPromServer(tlsConfig *api.PromTLSConf, server *http.Server) {
+	logrus.Debugf("entering startServer")
```

Comment:
nit: doc mismatch `StartServer` => `StartPromServer`, and debug log `startServer` => `StartPromServer`

Reply:
done